### ISSUE TITLE
Freeze after launch Gephi

### ISSUE THREAD

**Issue by Dady58** [gephi/gephi#1931]
## Expected Behavior
On the " Welcome Windows" Select and start a new Gephi project

## Current Behavior
The "Welcome to Gephi" window appears, but the application is frozen
Some Time a Windows "Authentication Required" is displayed, and after I click on the Cancel button, I can continue to use Gephi. 

## Possible Solution
Proxy lookup failed ?

## Steps to Reproduce

1.  Launch Gephi 
2. All modules is successfully loaded 
3. Try to create or open a project 
4. The "Welcome to Gephi" window appears, but the application is frozen
 

## Your Environment

* Version used: Gephi 0.9.2
![09-03-2018 09-32-23](https://user-images.githubusercontent.com/37298756/37296058-b9b02914-2611-11e8-8003-8c74889e5e02.jpg)
![12-03-2018 17-22-19](https://user-images.githubusercontent.com/37298756/37296060-bb4974ce-2611-11e8-841f-e34898075c3d.jpg)
[messages.log](https://github.com/gephi/gephi/files/1803634/messages.log)

* Java version:  jdk1.8.0_162 - x64
* Operating System: MS Windows 10 (Release 1709) or Windows 8.1

**Comment by Dady58** [comment]
Any help/solution on this issue are welcome ;-)

**Comment by eduramiba** [comment]
It seems your graphics card drivers is very old or inexistent, and also you are behind a proxy.

The auth required for proxy is not a problem if you ignore it, but you should install proper graphics card drivers in your system.

**Comment by Dady58** [comment]
We have the same problems with the proxy login and the freeze of Gephi after installing the latest graphics drivers on our HP laptops running Windows 8.1 or 10.
In the previous version 0.9, we did not have this problem.

**Comment by mbastian** [comment]
Thanks for the report. This hang is tied to the old engine's native-library startup probe (blocked by outdated/missing graphics drivers, or in some cases antivirus/corporate security intercepting the native-library check). Gephi 0.11 replaced this with a new visualization engine implementation that changes this startup path.

Could you retry on the latest 0.11.x release? If it still hangs, please open a fresh issue with your OS, driver version, and whether antivirus/corporate security software is active. Closing this for now as superseded by the new engine.
